src/main/java/duke/classes/Main.java

```diff
@@ -0,0 +1,30 @@
+package duke.classes;
+
+import java.io.IOException;
+
+import javafx.application.Application;
+import javafx.fxml.FXMLLoader;
+import javafx.scene.Scene;
+import javafx.scene.layout.AnchorPane;
+import javafx.stage.Stage;
+
+/**
+ * A GUI for Duke using FXML.
+ */
+public class Main extends Application {
```

Comment:
Should this class be named as Main? I personally feel a bit confused as you have classes MainWindow, Ui and Duke as well and I expected main() method to be in this class. Would you consider something like DukeGUI?